tests/pgshovel/streams/states.py

```diff
@@ -137,63 +152,29 @@ def test_require_different_publisher(message):
 
     require_different_publisher(
         Committed(
-            uuid.uuid1().bytes,  # change the publisher
-            batch_identifier,
+            publisher=uuid.uuid1().bytes,  # change the publisher
+            batch_identifier=batch_identifier,
         ),
+        0,
         message,
     )
 
     with pytest.raises(InvalidPublisher):
         require_different_publisher(
             Committed(
-                message.header.publisher,
-                batch_identifier,
+                publisher=message.header.publisher,
+                batch_identifier=batch_identifier,
             ),
+            0,
             message,
         )
 
 
-def test_stateful_validator():
-    Locked = namedtuple('Locked', '')
-    Unlocked = namedtuple('Unlocked', '')
-
-    validator = StatefulStreamValidator({
-        Unlocked: {
-            'coin': lambda state, event: Unlocked(),
-            'push': lambda state, event: Locked(),
-        },
-        Locked: {
-            'coin': lambda state, event: Unlocked(),
-            'push': lambda state, event: Locked(),
-        },
-    }, start=Locked())
-
-    assertions = (
-        ('push', Locked()),
-        ('coin', Unlocked()),
-        ('coin', Unlocked()),
-        ('push', Locked()),
-        ('push', Locked()),
-    )
-
-    inputs = (i[0] for i in assertions)
-    validated = validator(i[0] for i in assertions)
-    expected = (i[1] for i in assertions)
-    for input, (state, event), expected in itertools.izip(inputs, validated, expected):
-        assert input == event
-        assert state == expected
-
-    with pytest.raises(InvalidEventError):
-        next(validator(('kick',)))
-
-
-def test_stateful_validator_unhandled_starting_state():
-    events = range(5)
-    validator = StatefulStreamValidator({})
-    validated = validator(events)
+def test_stateful_validator_unhandled_starting_state(message):
+    validator = StatefulStreamValidator(lambda **kwargs: None, {})
 
     with pytest.raises(InvalidEventError):
```

Comment:
Totally agree. I think @tail brought this up as a point of confusion as well. They are used interchangeably here, but I think that "message" is probably the more appropriate term to use in general.